src/Services/SearchService.php

```diff
@@ -472,4 +475,30 @@ private function buildItemURL($itemData, $withVariationId = true): string
             )->toRelativeUrl($includeLanguage);
         }
     }
+
+    /**
+     * @param Request $request
```

Comment:
Only add things to the docblock, that aren't already part of the PHP native typings. `mixed` is fine, as it can not be typehinted with PHP 7.0, but `Request` is already typed by PHP.